spec/fortnox/api/repositories/examples/find_examples.rb

```diff
@@ -6,11 +9,15 @@
   end
 
   describe '.find' do
+    include_context 'JSONHelper'
+
     specify 'returns correct class' do
       expect( find_id_1.class ).to be described_class::MODEL
     end
 
     specify 'returns correct Customer' do
+      id_attribute_json = subject.options.unique_id
+      id_attribute = JSONHelper.new.convert_from_json(id_attribute_json)
```

Comment:
Great idea, but I could not get it to work. `convert_key_from_json` uses `@options`, which is set when the repository is initialized...